src/models/AuxiliaryChainRepository.ts

```diff
@@ -0,0 +1,181 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+/* eslint-disable class-methods-use-this */
+
+import {
+  DataTypes, Model, InitOptions, Op,
+} from 'sequelize';
+
+class AuxiliaryChainModel extends Model {}
+
+/**
+ * To be used for calling any methods which would change states of record(s) in Database.
+ */
+export interface AuxiliaryChainAttributes {
+  chainId: number;
+  originChainName: string;
+  ostGatewayAddress: string;
+  ostCoGatewayAddress: string;
+  anchorAddress: string;
+  coAnchorAddress: string;
+  lastProcessedBlockNumber?: number;
+  lastOriginBlockHeight?: number;
```

Comment:
The types should be BigNumber too. `BN`.
Can you please check all the attributes of already merged models.